apps/patient/src/components/BrandedPharmacyCard.tsx

```diff
@@ -18,14 +17,13 @@ import capsulePharmacyIdLookup from '../data/capsulePharmacyIds.json';
 
 interface Props {
   pharmacyId: string;
-  selectedId: string;
-  handleSelect: (id: string) => void;
+  selectedId?: string;
```

Comment:
This should be `string | undefined`. You presumably always want the parent to send it down?

Reply:
status page doesn't use the selectedId stuff since the pharmacy is just presentational, so i just made it optional